### PR TITLE
Documentation around how to add documentation

### PR DESCRIPTION
**Pull request by michaelamoako**
Additional documentation for modifying, or adding docs

### CODE REVIEW

**Comment by michaelamoako** [comment]
fairness_in_machinelearning can be ignored as that is unchanged from the Construct Validity PR

File: docs/contributor_guide/contributing_documentation.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Contributing documentation
+------------------------------
+
+Documentation is formatted in restructured text (ReST). To contribute, make sure to install sphinx and its
+add-ons by running :code:`python scripts/install_requirements.py --pinned False` in the repository
+root directory. You may also need to `install Pandoc <https://pandoc.org/installing.html>`_. You can contribute
+updates to existing documentation by navigating to the relevant part of the repository, and editing the restructured
+text files corresponding to your updates.
+
+To build the webpage run the following command from the repository root
```
Comment (romanlutz) [inline docs/contributor_guide/contributing_documentation.rst:12]: This is already documented: https://fairlearn.org/main/contributor_guide/development_process.html#building-the-website

File: docs/user_guide/fairness_in_machine_learning.rst [hunk 2]
```diff
@@ -39,6 +39,80 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Concept Glossary
```
Comment (romanlutz) [inline docs/user_guide/fairness_in_machine_learning.rst:42]: this is from another PR (?)

**Comment by michaelamoako** [comment]
Part of what I wanted to add was the command to see the rendering of the created doc:

start docs/_build/html/user_guide/_____.html

Where ____ is the name of the doc which was to be rendered in the user guide. Is this a change that you think makes sense to add to the doc (both for contributing example notebooks, and for contributing docs). Since this step was non-obvious? @romanlutz

**Comment by michaelamoako** [comment]
Pending: @romanlutz to merge the info from development_process to finalize the above

File: docs/contributor_guide/contributing_documentation.rst (hunk 1 above)
Comment (romanlutz) [reply docs/contributor_guide/contributing_documentation.rst:25]: Merged it with the existing docs after discussing with @michaelamoako :-)

**Review by romanlutz — APPROVED** [review]
As discussed with @michaelamoako I merged in the existing docs so that we have single page that talks about this. I really like that we have a dedicated page for this now thanks to @michaelamoako ! @fairlearn/fairlearn-maintainers can someone else also approve/merge this?

**Review by hildeweerts — APPROVED** [review]

**Pull request merged**